core/tests/multiproc.py

```diff
@@ -6,40 +6,38 @@
 # File to disk
 n = 0
 def addinfo(fr):
-	global n
-	if fr.type != core.G3FrameType.Timepoint:
-		return
-	fr['time'] = core.G3Time(int(time.time()*core.G3Units.s))
-	fr['count'] = n
-	n += 1
+    global n
+    if fr.type != core.G3FrameType.Timepoint:
+        return
+    fr['time'] = core.G3Time(int(time.time()*core.G3Units.s))
+    fr['count'] = n
+    n += 1
 m = 0
```

Comment:
Can you please undo the whitespace changes in this file?

Reply:
as you wish...